apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/StringUtil.java

```diff
@@ -23,6 +23,10 @@ public static boolean isEmpty(String str) {
         return str == null || str.length() == 0;
     }
 
+    public static boolean isBlank(String str) {
```

Comment:
There is already `StringUtil#isEmpty`. This is duplicated.

Reply:
> There is already `StringUtil#isEmpty`. This is duplicated.

They're different (trivial though) IMO, `StringUtil.isEmpty("  \t") == false` while `StringUtil.isBlank("  \t") == true`

Reply:
I highly doubt this is the purpose of creating this API..

Reply:
Maybe I should do String#trim on GrapthQL resolver?

Reply:
Why do you need a trim? Who will send an empty string?

Reply:
> Why do you need a trim? Who will send an empty string?

@wu-sheng As long as the string is typed by the users, it's totally possible that it will include blank characters, or even unprintable/invisible characters, we're suffering from this because the users copy-and-pastes from else where (especially Windows users, `\r` is painful), another solution is to trim and validate the input  in the frontend, but we may have other user interface, CLI I mean

Reply:
If that is the concern, `trim` the data in the GraphQL service level, this kind of blank issue clearly is related to humans, not to codes.